custom_components/tuya_local/devices/smart_p01_socket.yaml

```diff
@@ -0,0 +1,110 @@
+name: Smart P01 Socket

```

Comment:
This top level name should be without branding, quite likely this same device is sold in different markets or by different sellers on aliexpress under different brands.

Reply:
Usual terminology is smartplug